modules/c++/math.linear/wscript

```diff
@@ -1,7 +1,7 @@
 NAME            = 'math.linear'
 MAINTAINER      = 'jmrandol@users.sourceforge.net'
 VERSION         = '0.2'
-MODULE_DEPS     = 'sys mem'
+MODULE_DEPS     = 'sys mem types algs'
```

Comment:
math.linear can't depend on algs because algs isn't out on CODA-OSS... do you actually need anything from algs?